logging/fb/viya-tracing.conf

```diff
@@ -0,0 +1,12 @@
+# Version 1.9.9
```

Comment:
what does this version number represent?

Reply:
I actually meant to ask you that because I copied this from viya-parsers.conf but I meant to delete it, which I will do. Thanks for catching it.